.changes/next-release/Feature-a3aaba22-c8b1-4dd1-9064-f27aad72bc25.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
+	"description": "CodeCatalyst: Add developer setting to override the API configuration."
```

Comment:
Thanks for adding a changelog, that's always a good instinct. But in this case we probably shouldn't, because developer features are not supported and not relevant to most customers.